README.adoc

```diff
@@ -122,13 +122,28 @@ IMPORTANT: The `id` used in your `settings.xml` *must* be `github-sirius-emfjson
 1. Clone this repository
 2. Build the frontend packages:
 +
+[source,sh]
 ----
 npm ci
 npx turbo run build
 ----
 +
 
 NOTE: In order to run tests, use `npx turbo run coverage`
++
+
+[WARNING]
+====
+To build the package `@eclipse-sirius/sirius-components-diagrams` the `mkdir` command is required.
+
+For windows users, according to the link:https://docs.npmjs.com/cli/v8/using-npm/config#script-shell[npm documentation] the default value for script-shell configuration on windows is `cmd.exe` which does not support the `mkdir` command.
+
+You can set the `script-shell` configuration with the following command:
+[source,sh]
+----
+npm config set script-shell "path\\to\\bash.exe"
```

Comment:
I'm not a Windows user, but I don't think bash is available by default (it might be in recent versions, I don't know). Did we mention somewhere above that "git bash" must be installed?

Reply:
Done